src/etc/rust-lldb

```diff
@@ -10,7 +10,7 @@ host=$(rustc -vV | sed -n -e 's/^host: //p')
 RUSTC_SYSROOT=$(rustc --print sysroot)
 RUST_LLDB="$RUSTC_SYSROOT/lib/rustlib/$host/bin/lldb"
 
-lldb="lldb"
+lldb=lldb
```

Comment:
Can you say more about this change? I don't understand why it is needed...

Reply:
Maybe just leftover from the last two commits? We should squash regardless, but would be good to avoid spurious changes.

r=me with this dropped + commits squashed.

Reply:
I'll squash and clean up though you'd squash on merge anyway. But I messed up with some local testing and then tried to cleanup

Reply:
All squashed now